### ISSUE TITLE
CSE不支持多war包并行启动吗？

### ISSUE THREAD

**Issue by zhufeizzz** [apache/servicecomb-java-chassis#1568]
我们的Tomcat里面有多个war包，设置了并行启动，结果发现服务中心Address的urlPrefix串了。

跟踪了一下CSE的代码发现这个值的上下文是通过System.property传递的，这也太坑爹了吧。。。除了改成串行还有别的办法没？

ServletRestTransport
``` 
  @Override
  public boolean init() {
    String urlPrefix = System.getProperty(Const.URL_PREFIX);
    Map<String, String> queryMap = new HashMap<>();
    if (!StringUtils.isEmpty(urlPrefix)) {
      queryMap.put(Const.URL_PREFIX, urlPrefix);
    }

    String listenAddress = ServletConfig.getLocalServerAddress();
    setListenAddressWithoutSchema(listenAddress, queryMap);

    restClient = RestTransportClientManager.INSTANCE.getRestClient();
    return true;
  }
```

**Comment by liubao68** [comment]
java-chassis is not tested to deploy mutiple microservices in one process (or web container) and it's not recommended to do this. However, if you'd like to add some patch for this is welcome.

**Comment by WillemJiang** [comment]
It's not a good practice to host multiple microservices into one JVM.

**Comment by liubao68** [comment]
I crerated [an PR to fix this problem](https://github.com/apache/servicecomb-java-chassis/pull/1727), and will be available in 2.0.2
